website/client/src/components/header/notificationsDropdown.vue

```diff
@@ -167,9 +167,9 @@ export default {
     CARD_RECEIVED,
     NEW_INBOX_MESSAGE,
     NEW_CHAT_MESSAGE,
-    ACHIEVEMENT_JUST_ADD_WATER,
+    // ACHIEVEMENT_JUST_ADD_WATER,
     ACHIEVEMENT_LOST_MASTERCLASSER,
-    ACHIEVEMENT_MIND_OVER_MATTER,
+    // ACHIEVEMENT_MIND_OVER_MATTER,
```

Comment:
Same here, these can be deletions rather than comments

Reply:
deleted!